rice-framework/krad-sampleapp/web/src/it/java/org/kuali/rice/krad/labs/LabsEnterKeySupportTables2Aft.java

```diff
@@ -39,11 +39,7 @@ protected void navigate() throws Exception {
     }
 
     protected void testDemoEnterKeySupportTables2() throws InterruptedException {
-    	waitAndTypeByXpath("//table/tbody/tr/td[2]/div/input","1");
-    	waitAndTypeByXpath("//table/tbody/tr/td[3]/div/input","1");
-    	pressEnterByXpath("//table/tbody/tr/td[3]/div/input");
-    	pressEnterByXpath("//table/tbody/tr[2]/td[3]/div/input");
-    	acceptAlert();
+    	pressEnterByXpath("(//table/tbody/tr/td[3]/div/input)[1]");
```

Comment:
I don't think this would be a very useful test.  While the enter key is pressed there is no check that the enter key actually does something. 

I think we want something like this.  Correct the first three x-paths so that only the field in the add line is selected:
   waitAndTypeByXpath("//table/tbody/tr[1]/td[2]/div/input","1");
   waitAndTypeByXpath("//table/tbody/tr[1]/td[3]/div/input","1");
   pressEnterByXpath("//table/tbody/tr[3]/td[3]/div/input");
So here we added 1s into the fields and used the enter key to add a new row.  Add a check that verifies the row with the 1s has been added.  We know that the most recent row added is on the top. So that can be used for the x-path of that check if needed.

The next two lines work as they are supposed to.  The enter key is pressed in field 2 of the first result row, which causes an alert box to be displayed. acceptAlert() will fail if there is no alert box.
  pressEnterByXpath("//table/tbody/tr[2]/td[3]/div/input");
  acceptAlert();